xlive/Blam/Engine/game/players.h

```diff
@@ -81,6 +81,16 @@ struct s_player_profile_traits
 };
 ASSERT_STRUCT_SIZE(s_player_profile_traits, 16);
 
+#pragma pack(push, 1)
```

Comment:
remove pragma pack

Reply:
IIRC there's something funky going on with this header file, cause without this pack the following assert fails.

Reply:
fixed